game-loop/src/test/java/com/iluwatar/gameloop/VariableStepGameLoopTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * The MIT License
+ * Copyright © 2014-2019 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.gameloop;
+
+import java.lang.reflect.InvocationTargetException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * VariableStepGameLoop unit test class.
+ */
+public class VariableStepGameLoopTest {
+
+  private GameLoop gameLoop;
+
+  @Before
+  public void setup() {
+    gameLoop = new VariableStepGameLoop();
+  }
+
+  @After
+  public void tearDown() {
+    gameLoop = null;
+  }
+
+  @Test
+  public void testUpdate() {
+    try {
+      var method = VariableStepGameLoop.class.getDeclaredMethod("update", Long.class);
+      method.setAccessible(true);
+      method.invoke(gameLoop, 20L);
```

Comment:
I don't think it's good practise use reflection to make methods accessible. Maybe we could just make the method public by default? Or if that's a problem from design perspective maybe inherit from VariableStepGameLoop in this test and override the method visibility. The same issue is present in other tests too.

Reply:
Fixed in commit de763fbc4f138c37212e81e61710c8828e5f43cc